src/templates/Navigator/MainNavigatorComponent/Readme.md

```diff
@@ -0,0 +1,5 @@
+MainNavigatorComponent Example:
+
+    <MainNavigatorComponent>
```

Comment:
yeah, i see the complication.  a couple suggestions for how the API could look

```javascript
import { Navigator } from 'athenaeum';
import { NavigatorBody } from 'athenaeum';
import { NavigatorSidebar } from 'athenaeum';

export default () => (
  <Navigator>
    <NavigatorBody>
      { /* stuff here */ }
    </NavigatorBody>
    <NavigatorSidebar>
      { /* stuff here */ }
    </NavigatorSidebar>
  </Navigator>
);
```

or perhaps one step better is to group the related components like so:

```javascript
import { Navigator } from 'athenaeum';

export default () => (
  <Navigator>
    <Navigator.Body>
      { /* stuff here */ }
    </Navigator.Body>
    <Navigator.Sidebar>
      { /* stuff here */ }
    </Navigator.Sidebar>
  </Navigator>
);
```

whatchu thank?